post-scan-actions/fss-to-securityhub/template.yaml

```diff
@@ -0,0 +1,106 @@
+Parameters:
+  snsScanResultTopicArn:
+    Type: String
+    Description: FSS Results SNS Topic ARN.
+  LambdaZipBucket:
+    Type: String
+    Description: S3 bucket for lambda zip. If your have it under s3://bucket/path/to/zip.zip, enter 'bucket' here. 
+  LambdaZipKey:
+    Type: String
+    Description: S3 key for lambda zip. If your have it under s3://bucket/path/to/zip.zip, enter 'path/to/zip.zip' here. 
+Resources:
+  sentToSecurityHubLambdaFunctionServiceRole58D0F596:
```

Comment:
What are those `58D0F596` postfix for? CloudFormation would generate some random string if the don't specify the name property.